scripts/ldap/setup_ldap.sh

```diff
@@ -94,8 +94,12 @@ rm -rf /etc/openldap/slapd.d/cn\=config/olcDatabase\=\{2\}mdb.ldif
 if [[ $forceclean == true ]]
 then
   rm -rf /var/lib/ldap/*
+  yum remove -y symas-openldap-clients symas-openldap-servers
```

Comment:
for force clean , we dont remove pkg installation ..
we need to handle only data deletion

Reply:
done